.github/CODEOWNERS

```diff
@@ -24,7 +24,7 @@
 /charts/prometheus-pingdom-exporter/ @monotek @rpahli
 /charts/prometheus-postgres-exporter/ @gianrubio @zanhsieh
 /charts/prometheus-pushgateway/ @cstaud @gianrubio
-/charts/prometheus-rabbitmq-exporter/ @juanchimienti
+/charts/prometheus-rabbitmq-exporter/ @juanchimienti @iamabhishek-dubey
```

Comment:
This should be sorted. There is a script in this repo you can run that should do this for you locally based on maintainers listed in the charts